api/lib/domain/usecases/get-correction-for-answer.js

```diff
@@ -27,3 +31,20 @@ function _validateCorrectionIsAccessible(assessment, userId, answerId) {
     throw new AssessmentNotCompletedError();
   }
 }
+
+function _addSaveInfoToTutorials(savedTutorialIds) {
+  return (tutorial) => {
+    tutorial.isSaved = _.includes(savedTutorialIds, tutorial.id);
+    return tutorial;
+  };
+}
+
+async function _addTutorialSaveInfoToCorrection(userTutorialRepository, correction, userId) {
+  const savedTutorials = await userTutorialRepository.find({ userId });
+  const savedTutorialIds = _.map(savedTutorials, 'tutorialId');
+
+  correction.tutorials = correction.tutorials.map(_addSaveInfoToTutorials(savedTutorialIds));
```

Comment:
Comme les objets sont mutés, ce `map` est un `forEach` déguisé. Utiliser plutôt `forEach`.

```suggestion
  correction.tutorials.forEach(_addSaveInfoToTutorials(savedTutorialIds));
```